app/controllers/bookings_controller.rb

```diff
@@ -1,7 +1,11 @@
 class BookingsController < ApplicationController
 
   def index
-    @bookings = Booking.where(user_id: current_user.id)
+    if params[:status]
```

Comment:
We don't allow nill/null status on db so this will always be true. I think what we need here is `if current_user.role == "renter"`.

Reply:
In my original pundit there is:
 `user_not_authorized if current_user.provider?`

Also Booking.scope is no longer here so it will crash I think :(

Maybe we can amend as per below?

`@bookings = policy_scope(Booking).where(status: params[:status], user_id: current_user.id)
   user_not_authorized if current_user.provider?`

Reply:
ooooh nevermind we need params[:status] because of your filters right! but we still need policy.scope I guess T_T